packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -352,6 +363,7 @@ export const JETPACK_MONTHLY_PLANS = <const>[
 	PLAN_JETPACK_COMPLETE_MONTHLY,
 ];
 export const JETPACK_RESET_PLANS = <const>[
+	...JETPACK_STARTER_PLANS,
```

Comment:
That makes total sense. I added it because the golden token was also there and I was afraid I was missing something.

Reply:
Just checked it and it seems like `JETPACK_RESET_PLANS` is not only for that check, it's a constant for plans that were added after the reset, and some `type`s depend on that as well.

Reply:
My take on this would be to leave it in `JETPACK_RESET_PLANS` to be consistent with other plans. If this constant is not used - we can declutter it as a separate PR in the future as part of jetpack excellence.